src/main/java/com/desiato/puresynth/services/UserService.java

```diff
@@ -40,6 +49,13 @@ public void deleteUser(Long id) {
         userRepository.deleteById(id);
     }
 
+    @Transactional
+    public void deleteUserSessions(Long userId) {
```

Comment:
I think this should be part of `SessionService` or maybe `AuthenticationService`.

Reply:
moved to AuthenticationService